webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -32,15 +31,15 @@ const AuditCard = ({ hostName }) => {
     }
   };
 
-  const url = hostName && foremanUrl(`/api/audits?search=host+%3D+${hostName}`);
+  const url = hostName && `/api/audits?search=host+%3D+${hostName}`;
   const {
     response: { results },
   } = useAPI('get', url);
   return (
     <Card isHoverable>
       <CardTitle>
         {__('Recent Audits')}{' '}
-        <a href={foremanUrl(`/audits?search=host+%3D+${hostName}`)}>
+        <a href={`/audits?search=host+%3D+${hostName}`}>
```

Comment:
IMHO this still needs a helper to escape the hostname.

Reply:
Right, although this will be changed anyways in https://github.com/theforeman/foreman/pull/8598 - so i'll update this pr if the RFC is accepted once that is merged.